src/main/java/me/totalfreedom/totalfreedommod/command/Command_gsay.java

```diff
@@ -7,7 +7,7 @@
 import org.bukkit.entity.Player;
```

Comment:
This entire command needs to just be deleted...

Reply:
i thought gsay was deleted but idk

Reply:
deleted it, will be commited in a bit